pkgs/servers/http/hiawatha/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, cmake,
+pkgs, libxslt, zlib, libxml2,
```

Comment:
you don't need to go through pkgs, `openssl` is available in the callPackage scope.